Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -415,9 +439,9 @@
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Checkpoints were coded and added to Marble Blast Platinum two weeks before its first full release to the public.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: The original error message noise was in fact Matan screaming 'Spy', referring to Spy47 who originally coded the Online section.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Due to a programming oversight, Marble Blast Platinum used to be able to delete any file or folder from a user's computer, including system files and folders.";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Hypercube was an exclusive on Marble Blast Gold for the XBOX. Find it in Ultra's levels as Advanced 21. It retains the Gold marble size and disables the radar.";
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Before the Xbox section was added to Marble Blast Gold in PlatinumQuest, Hypercube used to be tacked on to the end of Marble Blast Ultra's Advanced section instead, with Gold marble size and no Blast, but Ultra everything else.";
```

Comment:
Currently it still is and it won't be moved out for this update. Let's revert this for now, until it gets moved out later